Sources/Extensions/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -46,3 +46,20 @@ public extension Optional {
 	}
 	
 }
+
+// MARK: - Operators
+infix operator ??= : AssignmentPrecedence
+
+/// SwifterSwift: If `rhs` is not `nil`, assign it to `lhs`.
```

Comment:
I think **Assign an optional value to a variable only if the value is not nil** makes more sense here :)